src/main/kotlin/org/jitsi/nlj/dtls/TlsClientImpl.kt

```diff
@@ -122,7 +137,7 @@ class TlsClientImpl : DefaultTlsClient() {
     override fun getMinimumVersion(): ProtocolVersion = ProtocolVersion.DTLSv10;
 
     override fun notifyAlertRaised(alertLevel: Short, alertDescription: Short, message: String?, cause: Throwable?) {
-        val stack = with(StringBuffer()) {
+        with(StringBuffer()) {
```

Comment:
hm, i think my intention was to create the stack and log it here, but looks like it wasn't.  should we re-add `stack` and then log it?  otherwise this is basically doing nothing